debian/control

```diff
@@ -8,7 +8,9 @@ Build-Depends: autotools-dev,
                gobject-introspection (>= 0.9.7),
                gtk-doc-tools (>= 1.18),
                libgirepository1.0-dev,
-               libxapian-1.3-dev
+               libxapian-1.3-dev,
+               meson (>= 0.42.0),
```

Comment:
You can probably remove the build-dep on `autotools-dev` too, right?

Reply:
And it's gone…